gc/base/MemoryPoolAddressOrderedList.cpp

```diff
@@ -47,6 +47,21 @@
 #include "MemcheckWrapper.hpp"
 #endif /* defined(OMR_VALGRIND_MEMCHECK) */
 
+#include "HeapRegionManager.hpp"
+#include "HeapRegionDescriptor.hpp"
+
+/**
+ * Called when SATB barrier is enabled/disabled. We use this to set the TLH alignment base.
+ */
+void
+concurrentSATBToggled(J9HookInterface** hook, uintptr_t eventNum, void* eventData, void* userData)
+{
+	MM_ConcurrentSATBEvent* event = (MM_ConcurrentSATBEvent*)eventData;
+	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+
+	((MM_MemoryPoolAddressOrderedList*)userData)->setParallelGCAlignment(env, event->satbEnabled);
+}
```

Comment:
a few 'star' corrections
please, scan the rest of the new code one more time, I think there are still a couple of spots not following the standard

```
concurrentSATBToggled(J9HookInterface **hook, uintptr_t eventNum, void *eventData, void *userData)
{
	MM_ConcurrentSATBEvent *event = (MM_ConcurrentSATBEvent *)eventData;
	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(event->currentThread);

	((MM_MemoryPoolAddressOrderedList *)userData)->setParallelGCAlignment(env, event->satbEnabled);
}
```